packages/fuel-gauge/test/fixtures/forc-projects/script-raw-slice/src/main.sw

```diff
@@ -12,7 +12,7 @@ struct Wrapper<T> {
 }
 
 fn validate_raw_slice(input: raw_slice) {
-    let vec: Vec<u64> = Vec::from(input);
+    let vec: Vec<u8> = Vec::from(input);
```

Comment:
`raw_slice` contains `u8` bytes not `u64`, so this test needed updated. This was updated initially in #1673 but only now can we compile the`raw_slice` in script test.

Reply:
Is this the issue you mentioned, or something else?

Just for clarity, I was referring to a [GH issue](https://github.com/FuelLabs/fuels-ts/pull/2566#issuecomment-2181510236), but I only found this one for you:
 - https://github.com/FuelLabs/sway/issues/5860

Reply:
Apologies for the confusion, this was an internal report. This is now tracked here:
- https://github.com/FuelLabs/sway/issues/6170